parser/src/lexer.rs

```diff
@@ -494,6 +497,55 @@ where
         Ok(())
     }
 
+    /// Lex a single magic command.
+    fn lex_magic_command(&mut self, kind: MagicKind) -> LexResult {
```

Comment:
Is it possible to parenthesize magic commands or do we intentionally exclude this from scope?

Reply:
No, it isn't possible to parenthesize them:

```
from IPython.core.inputtransformer2 import TransformerManager

t = TransformerManager()

t.transform_cell("(%matplotlib --inline)")
# '(%matplotlib --inline)\n'

# ...other escape commands results in the same output
```

_The code can be run as is in the IPython REPL_